cityofaustin/atd-moped
# Add subproject relationship between projects

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9323
https://github.com/cityofaustin/atd-data-tech/issues/9324
https://github.com/cityofaustin/atd-data-tech/issues/9310


## Testing
**URL to test:** 
https://moped-test.austinmobility.io/moped/projects/156

**Steps to test:**

1. Navigate to a project. As long as the project is not already a subproject/child project, you will see the Subprojects table. 
2. click the button to add a subproject
3. No parent projects nor already designated child projects should appear in the dropdown. Neither should the project you are viewing. 
4. Add a child project. 
5. Click child project link to confirm it takes you to that project's summary page. (There should not be a subproject table)
6. Delete a subproject. 

---
#### Ship list
